spec/services/fetch_projects_spec.rb

```diff
@@ -34,6 +34,31 @@
       # Team members and projects are linked together
       expect(team_member.projects).to include(project)
     end
+
+    it "recreates assignments" do
+      team_member = TeamMember.create(first_name: "Jay", last_name: "Smith", tenk_id: 387517)
+      old_project = Project.new(name: 'A Project', tenk_id: 404)
+      team_member.assignments.create(project: old_project)
+      expect(team_member.projects).to include(old_project)
+
+      described_class.new.call
+
+      expect(team_member.projects).not_to include(old_project)
+    end
+
+    it "skips over ghost assignments" do
```

Comment:
What are "ghost assignments"?

Reply:
Sometimes there is an `assignable_id` attached to the `user`, but trying to retrieve the project corresponding to that id results in a Not Found error.

Reply:
Can we rename the test to specify that, maybe?

Reply:
Sure, I wanted to avoid terminology such as "orphaned" but also have the description fit in one line.

Reply:
`skips over assignments that don't exist`?

Reply:
I've updated the test to add `context`, ready for another look